src/components/taskDetails/ProgressUpdateCard/LatestProgressUpdateCardPresentation.tsx

```diff
@@ -79,6 +91,48 @@ export default function LatestProgressUpdateCardPresentation({
                             </span>
                         </span>
                     </Tooltip>
+                    {isDevMode &&
+                        (username === '' ? (
+                            <img
+                                src={userProfileImageUrl}
+                                alt={'Avatar'}
+                                data-testid="latest-progress-update-card-profile-picture"
+                                className={
+                                    styles[
+                                        'latest-progress-update-card__profile-picture'
+                                    ]
+                                }
+                            />
+                        ) : (
+                            <Tooltip
+                                content={username}
+                                tooltipPosition={{
+                                    top: '-2.2rem',
+                                    right: '-3.8rem',
+                                    width: '10rem',
+                                }}
+                            >
+                                <a
+                                    href={`${USER_MANAGEMENT_URL}?username=${username}`}
+                                    className={
+                                        styles[
+                                            'latest-progress-update-card__user-info-link'
+                                        ]
+                                    }
+                                >
```

Comment:
Should we open the user details in a new tab when the avatar is clicked to let users retain their current context?

Reply:
I’ve decided to keep it in the same tab to maintain a smoother flow, but if you prefer, we can also open it in a new tab to retain the current context. Let me know your thoughts!

Reply:
Was any particular flow discussed during the design doc preparation? If not, I think it's fine to follow the current approach of keeping it in the same tab for now.

Reply:
No specific flow was discussed in the design doc regarding this.